xml/docu_styleguide.language.xml

```diff
@@ -7,6 +7,14 @@
 <sect1 xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="sec-language">
  <title>Language</title>
 
+ <para>
+  We write our documentation in English, using American English as default
+  variant. This mostly affects spelling of terms. If spellings differences exist
+  between American and British English, use the American English variant. For
+  example, use <literal>internationalization</literal> instead of
+  <literal>internationalisation</literal>.
+ </para>
+
```

Comment:
1. We mention the use of AmE below already, although in the same sentence that we also mention M-W. I think making it clearer that we use AmE earlier as done in this commit makes sense though.  This information should just be deduped a bit.

2. The -ise/-ize thing I recently learned is actually a language change that was largely driven by the MS Word spell checker -- before the days of MS Word, BrE speakers apparently used -ise and -ize interchangeably, but MS's spell checker corrected that away. Maybe it would be better to just say we use the -ize variants where they exist. :)

Reply:
Updated according to your proposal.